docs/src/comparisons.md

```
@@ -0,0 +1,57 @@
+# Comparisons with Other Solutions
+
+This document compares KubeElasti with other popular serverless and scale-to-zero solutions in the Kubernetes ecosystem.
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)